.github/workflows/web.yml

```diff
@@ -8,10 +8,12 @@ on:
 
 jobs:
     build:
-        if: github.actor == 'OSBotify'
         runs-on: ubuntu-16.04
 
         steps:
+            - name: Echo actor
```

Comment:
Should we echo this on all of them since once was working and the other wasn't?

Reply:
Updated!